numpy/core/tests/test_cpu_features.py

```diff
@@ -1,8 +1,53 @@
 import sys, platform, re, pytest
-
-from numpy.testing import assert_equal
 from numpy.core._multiarray_umath import __cpu_features__
 
+def assert_equal(actual, desired, fname):
```

Comment:
the name is confusing since it is specific to cpu features, perhaps `assert_features_equal` ?
```suggestion
def assert_features_equal(actual, desired, fname):
```

Reply:
Yes, it would be better. thank you.